etc/aws/template

```diff
@@ -153,7 +153,7 @@
                                 "            -v /:/host:ro",
                                 "            -v /tmp/home/core:/host/home/core",
                                 "            -i pachyderm/pfs",
-                                "            /go/bin/deploy -disk /dev/xvdf ",
+                                "            /go/src/github.com/pachyderm/pachyderm/etc/bin/launch-wrapper /go/bin/deploy -disk /dev/xvdf ",
```

Comment:
That's on purpose. AWS turns the next line in to a set of user defined args so we need a space there to make sure they get parsed correctly.